### DISCUSSION TITLE
Multiple Vaults support from Chrome Extention

### DISCUSSION THREAD

**Discussion by jonasbaldwin** [jgchristopher/obsidian-clipper#55] [Ideas]
First off, this is amazing! I wish I'd had this when I was still in college and doing research papers!

I do wish that I could have different configurations for different computers from the Chrome extension (while still syncing Chrome settings). I use a different vault for my work notes and my personal notes and while I could use the bookmarklet, I'd prefer to use the extension for both.

**Comment by jgchristopher** [comment]
I think this can be supported along side some of the changes I am currently working on, which would allow you to manage/configure multiple clippers.

**Reply by jonasbaldwin** [reply]
Awesome!
